givitsite/friendfeed/admin.py

```diff
@@ -1,3 +1,5 @@
 from django.contrib import admin
+from .models import FoundItem
 
 # Register your models here.
+admin.site.register(FoundItem)
```

Comment:
I see "no newline at end of flie" very often. Probably we need to have a way to avoid this.
What editor are you using?

Reply:
vscode. I saw your review on #13, I will fix it.